apps/native/app/src/ui/lib/card/notification-card.tsx

```diff
@@ -85,9 +85,7 @@ export function NotificationCard({
     <Host
       onPress={() => onPress(id)}
       underlayColor={
-        underlayColor ?? theme.isDark
-          ? theme.shade.shade400
-          : theme.color.blue100
+        underlayColor ?? theme.isDark ? theme.shade.shade400 : '#EBEBFA'
```

Comment:
What is this color? Should there be a color constant?

Reply:
I asked Daði the exact same question 😂 Apparently this is just a color that is only used for this underlayPress color, he wanted to keep it as is (used in one other place in the app) until we get more time to go over all the styles in the project.